stable_plugins/classical_ml/data_preparation/wu_palmer.py

```diff
@@ -490,6 +497,16 @@ def calculation_task(self, db_id: int) -> str:
         attributes,
         root_has_meaning_in_taxonomy,
     ) = load_input_parameters(db_id)
+    deserializer: Callable[[tuple[str, ...]], tuple[any, ...]] | None = None
+
+    with open_url(entities_metadata_url) as entities_metadata_file:
+        entities_metadata_list = list(
+            load_entities(entities_metadata_file, "application/json")
```

Comment:
use the get_mimetype function to guess the mimetype here instead of hardcoding json